Vungle/VungleInterstitialCustomEvent.m

```diff
@@ -213,4 +216,14 @@ - (NSString *)getPlacementID
     return self.placementId;
 }
 
+- (NSString *)getAdMarkup
```

Comment:
We might wanna create "parent" class in future? Just reminder. No need to change now ;-)